modules/extensions-xml-support/src/test/resources/module-documentation/module-documentation.xml

```diff
@@ -12,10 +12,12 @@
            http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
 
     <property name="aPropertyWithDoc" type="string" doc:description="Documentation for the property"/>
+    <property name="aHiddenPropertyWithDoc" type="string" password="true" doc:description="Documentation for the hidden property"/>
 
     <operation name="operation-with-doc" doc:description="Documentation for the operation">
         <parameters>
             <parameter name="paramWithDoc" type="string" doc:description="Documentation for the parameter" />
+            <parameter name="hiddenParamWithDoc" type="string" password="true" doc:description="Documentation for the hidden parameter" />
```

Comment:
Another thing. Review how this is approached in the SDK with annotations and if there are other types for parameters so we align the model of the XML with the model of the SDK if it makes sense.